src/SkynetServer/Services/MessageInjectionService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.EntityFrameworkCore;
+using SkynetServer.Database;
+using SkynetServer.Database.Entities;
+using SkynetServer.Model;
+using SkynetServer.Network;
+using SkynetServer.Network.Model;
+using SkynetServer.Network.Packets;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SkynetServer.Services
```

Comment:
[OT] Now, it not only doesn't syntax-highlight, but also no longer loads diffs [/OT]